lib/babel.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 
+var pkgDir = require('pkg-dir').sync;
 var debug = require('debug')('ava');
+var hasha = require('hasha');
+var cacha = require('cacha');
+var path = require('path');
 
 var opts = JSON.parse(process.argv[2]);
+var testPath = opts.file;
+
+var cache = cacha(path.join(pkgDir(path.dirname(testPath)), 'node_modules', '.cache', 'ava'));
```

Comment:
So, the cache is being stored in `node_modules/.cache/ava`, but [later down this file](https://github.com/sindresorhus/ava/pull/352/files#diff-c56ba0fef1bb510acefb4680eb31e9c6R151) versions from ava & babel-* packages are being included in the cache key.

That way, when users `rm -rf node_modules/.cache/ava`, they can be sure they removed all the cache. And they won't need to get the AVA version to remove `ava-0.8.0` (for example).